implementation_design.md

```diff
@@ -1,214 +1,69 @@
-    function finalizeWithdrawal(uint256 amount) external;
-
-    /// @notice Only the auctioneer can call this method. If there are only two distinct bids
-    ///         present for bidding on the upcoming round, the round can be deemed canceled by setting
-    ///         the express lane controller to the zero address.
-    function cancelUpcomingRound() external;
-
-    /// @notice Allows the upcoming round's express lane controller to delegate ownership of the express lane
-    ///         to a delegate address. Can only be called after an auction has resolved and before the upcoming
-    ///         round begins, and the sender must be the winner of the latest resolved auction. Will update
-    ///         the express lane controller for the upcoming round to the specified delegate address.
-    /// @param delegate the address to delegate the upcoming round to.
-    function delegateExpressLane(address delegate) external;
-
-    /// @notice Only the auctioneer can call this method, passing in the two highest bids.
-    ///         The auction contract will verify the signatures on these bids,
-    ///         and that both are backed by funds deposited in the auction contract.
-    ///         Then the auction contract will deduct the second-highest bid amount
-    ///         from the account of the highest bidder, and transfer those funds to
-    ///         an account designated by governance, or burn them if governance
-    ///         specifies that the proceeds are to be burned.
-    ///         auctions are resolved by the auctioneer before the end of a current round
-    ///         at some time T = AUCTION_CLOSING_SECONDS where T < ROUND_DURATION_SECONDS.
-    /// @param bid1 the first bid
-    /// @param bid2 the second bid
-    function resolveAuction(Bid calldata bid1, Bid calldata bid2) external;
-}
-```
+The latest Solidity interface for the express lane auction contract can be found [here](https://github.com/OffchainLabs/nitro-contracts/pull/214/files). The latest Go implementation is [here](https://github.com/OffchainLabs/nitro/compare/express-lane-timeboost?expand=1)
 
 # Offchain Components
 
 ## Auctioneer API
 
-The auctioneer must expose a new RPC namespace called `timeboost` and a RPC method called `timeboost_submitBid`
+The auctioneer must expose a new RPC namespace called `auctioneer` and a RPC method called `timeboost_submitBid`
 
-### `timeboost_submitBid`
+### `auctioneer_submitBid`
 
 ```json
 {
   "type": "object",
   "properties": {
-    "chain_id": {
-      "type": "uint64",
+    "chainId": {
+      "type": "bigInt",
       "description": "chain id of the target chain"
     },
-    "address": {
-      "type": "string",
-      "description": "0x-prefixed, 20-byte address of the bidder"
+    "expressLaneController": {
+      "type": "address",
+      "description": "hex string of the desired express lane controller address"
```

Comment:
Best to say whether or not this is 0x-prefixed. Same for the other "hex string" values below.